### ISSUE TITLE
my-list-selector-modal-view.component unusable on iOS

### ISSUE THREAD

**Issue by zurie** [NativeScript/template-master-detail-ng#86]
Using Emulator v11 and a real iPhone 7+ running iOS 11.0.2

when going into the "edit" section of the template and activating any option that would pop up the my-list-selector-modal-view.component   you see a modal that is blank.

There is no way to close the modal and no way to select any options.

this is a clean install using Nativescript Sidekick.

Also, you can use the CSS and HTML and force some stuff to display but I have not found what the fix is yet.

Edit:  It seems that if you "Rotate" your iphone landscape and back to portrait it fixes the modal in portrait.  If you tilt your phone to landscape it works 100% of the time.

Is this a bug with CSS or a bug with something else?

**Comment by zurie** [comment]
It looks like upgrading the built in components to the latest version fixes it, (angular 4.4.4)  I updated Telerik UI to 3.1.3 (from 3.1.1) and left Angular at 4.2.5 and it did not fix it, but going to angular 4.4.4 does.

**Comment by manoldonev** [comment]
Indeed it seems there is an issue with the modal scenario with iOS 11 and nativescript-telerik-ui@3.1.1 used in the Sidekick version of the template at the moment.

I would suggest you to either manually update nativescript-telerik-ui dependency to 3.1.4 in your app (and keep the Angular dependencies unmodified), or you can re-create your app from scratch from the `master` branch of this repo that has the fixed dependencies out-of-the box (you will be able to open the app in Sidekick after that):
```
tns create my-app-name --template https://github.com/NativeScript/template-master-detail-ng
```
